### PR TITLE
Support source generating values with Utf8 formatting/parsing

### PR DESCRIPTION
**Pull request by rabuckley**

### CODE REVIEW

**Review by frankbuckley — COMMENTED** [review]
We will need to adapt JsonSpanSerializableValueConverter

**Comment by rabuckley** [comment]
> JsonSpanSerializableValueConverter

There is already `JsonUtf8SpanSerializableValueConverter`, I just need to switch them over.

**Comment by frankbuckley** [comment]
> There is already `JsonUtf8SpanSerializableValueConverter`, I just need to switch them over.

Then, value implementer will have to remember which to use, even though not explicitly marking Utf8 support.

Might be better for JsonSpanSerializableValueConverter to do type check and switch strategy if it can use Utf8?

**Comment by frankbuckley** [comment]
> Might be better for JsonSpanSerializableValueConverter to do type check and switch strategy if it can use Utf8?

Although I think that might not be straightforward

**Comment by frankbuckley** [comment]
Stick with Plan A for now

**Comment by rabuckley** [comment]
There are more (i.e. the AsciiChars) but they're not values backed by another so I'll do them in a separate PR.

File: src/DSE.Open.Records.Abstractions/ClinicalConceptCode.cs [hunk 1]
```diff
@@ -18,11 +18,12 @@ namespace DSE.Open.Records;
 [EquatableValue]
 [JsonConverter(typeof(JsonSpanSerializableValueConverter<ClinicalConceptCode, long>))]
```
Comment (frankbuckley) [inline src/DSE.Open.Records.Abstractions/ClinicalConceptCode.cs:19]: Utf8?

And ConditionDiagnosisCode?

and HearingDiagnosisCode?

Comment (rabuckley) [reply src/DSE.Open.Records.Abstractions/ClinicalConceptCode.cs]: Done.

**Review by frankbuckley — APPROVED** [review]

**Pull request merged**